CMakeLists.txt

```diff
@@ -62,17 +61,18 @@ endif()
 include(compiler)
 include(os)
 
-if (64_BIT)
+if(64_BIT)
```

Comment:
Unrelated change

Reply:
I think it's fine though, as I edit the part just below.

Reply:
The part just below being `set(ARCH "x64")` which remains unchanged...

In general i think we should consider using a formatter for our cmake files in order to avoid this kind of stuff in regular commits...

Reply:
I agree, https://github.com/cheshirekow/cmake_format may be a good candidate.